### ISSUE TITLE
Check accessibility of summary report text

### ISSUE THREAD

**Issue by waldoj** [flexion/hud-disaster-data#90]
I'm concerned about the contrast level of this text ("State: Texas," etc.) on the Report page:

![contrast](https://user-images.githubusercontent.com/656758/35462275-44d8f092-02b9-11e8-9510-b201fddd3f68.png)

I'm not confident that it meets accessibility standards. Let's check on this.

**Comment by davehannon** [comment]
There are warnings from pa11y, but no errors.  It was my understanding that we only need to fix errors, so if we are going by that, we seem to be fine.

Having said that, if folks would like to see a change there, I would be open to implementing it.

Do you have any recommendations?

**Comment by waldoj** [comment]
AFAIK, pa11y isn't capable of checking the contrast of text against background images, only background _colors._ I'll check into this!

**Comment by waldoj** [comment]
I checked with [a contrast checker](https://webaim.org/resources/contrastchecker/), and it's OK-ish. The individual "pixels" provide a contrast ratio of 6.4:1, which fails WCAG AAA for normal text, but passes for large text. This text is large-ish. But taking a step back, and averaging not the center of each pixel but instead the average of each "tile," that's a 9:1 ratio, which is just fine.

Seems like it'll do the trick!
